docs/troubleshooting/updating-nitric.mdx

```diff
@@ -0,0 +1,13 @@
+## Updating Nitric
+
+If you're ready to upgrade your projects to a newer version of Nitric
+
+- Start by removing any versions of the membrane which are currently residing on Docker `docker system prune -a`
```

Comment:
A full prune may be a bit gratuitous, we might be able to get them to pull the latest version of the membrane explicitly using docker pull?

We're thinking once we have more stable versioning we will actually tie template versions to membrane versions as well, if it's explicitely specified it could save them the hassle of having to remove stale membranes.

Reply:
Sure  - I am happy to add this option in - 

- docker image rm nitricimages/membrane-local
- checkout membrane
- run make dev-docker or make aws-docker
- docker pull nitricimages/membrane-local

Anything missing?

Reply:
Added steps describing alternative to pruning.